taxonomies/categories.txt

```diff
@@ -102901,7 +102940,7 @@ fr:Palmiers salés
 nl:Gezouten vlinders, Gezouten palmiers
 
 <en:Crackers
-fr:Crêpes dentelle salées, Crêpes dentelles salées
+fr:Crêpes dentelles salées, Crêpes dentelles salées
```

Comment:
```suggestion
fr:Crêpes dentelle salées, Crêpes dentelles salées
```
Dentelle stays singular when crêpe is plural. French language makes no sense sometimes

Reply:
ok imolemented